probe-rs/src/architecture/arm/memory/adi_v5_memory_interface.rs

```diff
@@ -613,7 +741,7 @@ where
         Ok(())
     }
 
-    /// Write a block of 8bit words at `addr`.
+    /// Write a block of 8 bit words at `address`.
```

Comment:
I think this should also be changed, for the same reason as mentioned earlier. There are no 8 bit words.

Reply:
I am not sure I agree. We litterally have memory operations that are called like this. I see word used as the term for that everywhere.